great_expectations/datasource/data_connector/configured_asset_filesystem_data_connector.py

```diff
@@ -16,6 +16,17 @@
 
 
 class ConfiguredAssetFilesystemDataConnector(ConfiguredAssetFilePathDataConnector):
+    """
+    Extension of ConfiguredAssetFilePathDataConnector used to connect to Filesystem
+
+    The ConfiguredAssetFilesystemDataConnector is one of two classes (InferredAssetFilesystemDataConnector being the
+    other one) designed for connecting to filesystem-like data, more specifically files on disk. It connects to assets
```

Comment:
In this case not filesystem-like but specifically filesystem data.

Reply:
thank you for catching this